cognite_toolkit/_cdf_tk/commands/deploy.py

```diff
@@ -306,12 +306,23 @@ def to_create_changed_unchanged_triple(
 
         for item in resources:
             cdf_resource = cdf_resource_by_id.get(loader.get_id(item))
-            if cdf_resource and loader.are_equal(item, cdf_resource):
+            try:
+                are_equal = cdf_resource and loader.are_equal(item, cdf_resource)
+            except CogniteAPIError as e:
+                self.warn(
+                    MediumSeverityWarning(
+                        f"Failed to compare {loader.display_name} {loader.get_id(item)} for equality. Proceeding assuming not data in CDF. Error {e}."
+                    )
+                )
+                print(Panel(traceback.format_exc()))
+                are_equal = False
+            if are_equal:
```

Comment:
Accidental bug discovery. I was blocked by this so I solved it, and it was so small that I did not pull it out in its own PR.